app/task_routes.py

```diff
@@ -0,0 +1,110 @@
+from flask import Blueprint, jsonify, abort, make_response, request
+from app.models.task import Task
+from datetime import datetime
+from app import db
+import os
+from slack import WebClient
```

Comment:
To get this code to work, this project needs to install the `slack_sdk` package (described [here](https://slack.dev/python-slack-sdk/web/index.html), documentation [here](https://slack.dev/python-slack-sdk/api-docs/slack_sdk/)), and I'd expect this package to be listed in the `requirements.txt` file so anybody could install it.

Reply:
If you didn't want to use the `slack_sdk` package, another way you could approach the problem is to use the `requests` package to send a request, much like we did in the [Seven Wonders Activity](https://learn-2.galvanize.com/cohorts/3643/blocks/1310/content_files/requests-and-responses/problem-set-requests-with-python.md)